README.md

```diff
@@ -43,6 +44,8 @@ The following is a list of commands which are available for you to use once you
 |New-RsFolder|This command creates a new folder in the specified path.|
 |New-RsRestSession|This command creates a session object to be specified for all subsequent calls to the REST Endpoint.|
 |New-RsWebServiceProxy|This command creates a new Web Service Proxy which is used to interact with the SOAP Endpoint.|
+|New-RsSubscription|This command adds a new subscription to an existing report.|
```

Comment:
Please move this before New-RsWebServiceProxy.

Reply:
Done